lib/converter.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+const parser = require('./parser');
+const YAML = require('js-yaml');
+
+
+const formats = Object.freeze({
+    JSON: "json",
+    YAML: "yaml"
+});
+
+function convertToJson(content) {
+    let jsonObject = JSON.parse(content);
```

Comment:
Beware that JSON.parse can throw an Error. You might have to use try/catch here.